vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -0,0 +1,20 @@
+module SecureRooms
+
+  module AccessRules
+
+    class BaseRule
+
+      def self.call(user, card_reader, accounts, selected)
```

Comment:
hmmm we'd lose the ability to check the not-yet-implemented `direction` which might eventually be needed depending on the Occupancy code